ui/app/components/app/connected-accounts-list/index.scss

```diff
@@ -115,6 +115,18 @@
     i {
       padding-right: 8px;
       font-size: 18px;
+      color: #C4C4C4;
+    }
+  }
+
+  &__permissions {
+    max-height: 0px;
+    overflow: hidden;
+    height: auto;
+    transition: max-height 0.8s ease-out;
+    &--expanded {
+      // arbitrarily set hard coded value for effect to work
+      max-height: 100px;
```

Comment:
Can we use `unset` here? (Or maybe `inherit`/`initial`?)

Reply:
I tried using scaleY to animate the transition effect, but for some reason, it just doesn't work (i have to inspect element to even get it to take effect, do extensions not get access to GPU acceleration?) -- there are two approaches left for animating height:

1. animate max-height, leaving height to auto ( impossible to animate height directly)
2. use JavaScript to take measurements and animate it. Relatively heavy-handed but gets around setting a hardcoded value. I'm not opposed to going down that rabbit hole, though.

[source](https://css-tricks.com/using-css-transitions-auto-dimensions/)

Reply:
Gotcha, this is fine 😅